### PR TITLE
feat: add neutron_policy_server support

### PR DESCRIPTION
**Pull request by ricolin**
Depends-On https://github.com/vexxhost/neutron-policy-server/pull/1

### CODE REVIEW

**Review by mnaser — CHANGES_REQUESTED** [review]
please run this via uwsgi :)

**Review by mnaser — CHANGES_REQUESTED** [review]
@ricolin small revision, I think it will make life a lot easier, I think instead of building a new image, we should instead have the existing image install neutron-policy-server in the same image, therefore, it will allow us to co-install and share dependencies and re-use the same image.

I think we will need to finalize that release first though, let me get that through.

File: roles/neutron/defaults/main.yml [hunk 1]
```diff
@@ -27,3 +27,4 @@ neutron_ingress_annotations: {}
 
 # Enable dns integration
 neutron_designate_integration_enabled: false
+neutron_policy_server_integration_enabled: false
```
Comment (mnaser) [inline roles/neutron/defaults/main.yml:30]: Honestly, I see the usefulness of this for everyone, so let's enable it by default, and add a comment above it and an extra space.

**Review by mnaser — CHANGES_REQUESTED** [review]

**Comment by ricolin** [comment]
I think the last uwsgi/image change introduce the failed on CI. Debuging now

**Comment by mnaser** [comment]
regate

**Pull request merged**